llama_stack/apis/inference/client.py

```diff
@@ -127,19 +134,19 @@ async def run_mm_main(host: str, port: int, stream: bool, path: str):
     )
     cprint(f"User>{message.content}", "green")
     iterator = client.chat_completion(
-        model="Llama3.2-11B-Vision-Instruct",
+        model=model,
         messages=[message],
         stream=stream,
     )
     async for log in EventLogger().log(iterator):
         log.print()
 
 
-def main(host: str, port: int, stream: bool = True, mm: bool = False, file: str = None):
+def main(host: str, port: int, stream: bool = True, mm: bool = False, file: Optional[str] = None, model: Optional[str] = None):
```

Comment:
could you run `pre-commit`? make sure you install it first 

```
cd llama-stack
pip install pre-commit
pre-commit install
```

Then run `git commit -a --amend` (once this time) so it will run and run all the formatting lints.

Reply:
Oops, yes, it's about time I install that!

Reply:
done

Reply:
... and now with automation so you don't have to catch this yourself!  https://github.com/meta-llama/llama-stack/pull/176